src/app_service/service/wallet_account/service.nim

```diff
@@ -337,21 +339,28 @@ QtObject:
     ))
 
   proc onStartBuildingTokensTimer*(self: Service, response: string) {.slot.} =
+    if ((now().toTime().toUnix() - self.timerStartTimeInSeconds) * 1000 < CheckBalanceSlotExecuteIntervalInMilliseconds):
```

Comment:
@endulab you could just set `CheckBalanceSlotExecuteIntervalInMilliseconds` to `15 * 60` and in that case we won't need multiplying by `1000` here.

Reply:
Done